projects/cashmere/src/lib/table/table.md

```diff
@@ -129,6 +129,8 @@ If you are using the `HcTableDataSource` for your table's data source, provide t
 directive to the data source and it will automatically listen for sorting changes and change the
 order of data rendered by the table.
 
+The default sort order of values is as follows: "[Null]AaBbCc123".
```

Comment:
if you're doing `toLocaleLowerCase()`, isn't the sort order actually `ABC123` (case-insensitive)?  Wouldn't this be the sort order?
`aBc`
`AdE`

Reply:
Also, don't numbers usually sort before letters in alphabetical order?

Reply:
@joeskeen I'll make that change you suggested on the docs. The examples you're showing look correct to me. 

Yup typically numbers are sorted first, but the way the component is written they come second. Not my doing. I suppose we could change that too if we'd like